util/serial/Cargo.toml

```diff
@@ -20,6 +20,7 @@ prost = { version = "0.11", default-features = false, features = ["prost-derive"
 protobuf = { version = "2.27", optional = true }
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
 serde_cbor = { version = "0.11.1", default-features = false, features = ["alloc"] }
+serde_derive = "1.0.1"
```

Comment:
We shouldn't need to use `serde_derive` directly, if we have the `derive` feature turned on for serde.